docs/project-lifecycle.md

```diff
@@ -0,0 +1,123 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Project Lifecycle
+
+The term "project" within Hyperledger will refer to a
+collaborative endeavor to deliver a work item.
+There may be some projects that are intended to produce a document, such
+as a requirements or use cases document, a whitepaper, or analysis.
+Others may be to develop a new capability, or refactor (or remove) an
+existing capability for the Hyperledger technology releases. Such
+projects may take the form of a new component (e.g. a new repository) or
+may propose additions, deletions or changes to an existing
+repository(s).
+
+Many other open source initiatives leverage an incubation process for
+new work items, and this seems to have a desired effect of encouraging
+new ideas and tracks of work, while at the same time providing clear
+guidance to the broader community as to what is real and supported,
+versus what is still in the exploratory/experimental/developmental
+phases.
+
+Therefore Hyperledger has adopted a similar lifecycle process as
+follows:
+
+Projects are in one of five possible states:
+
+-   [*Proposal*](#proposal)
+-   [*Incubation*](#incubation)
+-   [*Active*](#active)
+-   [*Promoted Release*](#promoted-release)
+-   [*Deprecated*](#deprecated)
+-   [*End of Life*](#end-of-life)
+
+Projects may not necessarily move through those states in a linear way
+and may go through several iterations.
+
+# Proposal
+
+Project Proposals shall be submitted to the [TSC](https://www.hyperledger.org/about/leadership)
+for review, using [Proposal Template](https://wiki.hyperledger.org/display/TSC/Project+Proposals).
+Proposals that are approved shall enter into an *Incubation* state, unless
+they are of a refactoring nature, in which case they will be turned over
+to the relevant project maintainer(s) to handle as they deem fit.
+
+A Proposal must:
+
+-   have a clear description
+-   have a well-defined scope
+-   identify committed development resources
+-   identify initial maintainers
+-   be vendor neutral
+
+# Incubation
+
+Approved project proposals enter into *Incubation*. For new
+components/modules, a repository will be created under the
+[Hyperledger Github org](https://github.com/hyperledger)
+and optionally in [JIRA](https://jira.hyperledger.org), if requested. New
+features/capabilities should be handled through pull requests labeled
+with tags that identify the project and tag it as
+*incubator* (and will ideally be capable of being enabled/disabled with feature-flags).
+
+Projects in *Incubation* may overlap with one another.
+Entering *Incubation* is meant to be fairly easy to allow for
+community exploration of different ideas.
+
+Once a project qualifies to be declared *Active*, the
+*project*\'s maintainers can then vote to request a graduation
+review by the TSC.
+
+Projects seeking to graduate from *Incubation* must:
```

Comment:
Can we remove or revise this section? It seems to provide a case to say that incubation graduation consists of these 4 points whereas the Incubation Exit Criteria pages should be governing.  

Either drop it and pull the paragraph from 84 (the one linking to exit criteria) into this location or make the list sound non normative and have the paragraph at line 84 follow, with the paragraph about maybe never going active being the last in this section.

perhaps like this (drop must so it doesn't look like it's governing, and paragraph reorder):
```
Projects seeking to graduate from *Incubation*:
-   have a fully functional code base
-   have test coverage commensurate with other *Active*
    projects
-   have an active and diverse community of developers
-   have a history of releases that follow the *Active*
    release process

The criteria to exit *Incubation* are defined in the
[Incubation Exit Criteria](./project-incubation-exit.md) document.

Entering *Incubation* does not guarantee that the project will
eventually get to *Active* state. Projects may never get
to *Active* state.
```

Reply:
I like Danno's proposed text. Makes sense to me.

Reply:
OK; I made a slightly different change. [See the change here](https://github.com/hyperledger/tsc/compare/2980af22d45a20449437f2e786c378196ab76b5a..e0a2f3431cce029c0c0c9fb071732c1b192500ec)

Reply:
That works! Thanks.